packages/components/src/Table/Filters/BooleanFilter.js

```diff
@@ -0,0 +1,28 @@
+import Filter from './Filter';
+
+/**
+ * This is used to filter boolean values.
+ * Default behaviour is: true value is not filtered, false value is filtered.
+ * You can reverse this behaviour with the 'reverse' attribute set to true.
+ */
+export default class BooleanFilter extends Filter {
```

Comment:
Please avoid using class hierarchies, use composition, a class for a filter is total overkill, and local state make it hard to test and maintain.